src/theme/variables.scss

```diff
@@ -76,7 +76,7 @@
   --ion-color-light-tint: #f5f6f9;
 }
 
-@media (prefers-color-scheme: dark) {
+@media (prefers-color-scheme: light) {
```

Comment:
The comment below is talking about the dark colours so it would be good to leave the variable as dark as well